### ISSUE TITLE
Collecting multiple test files

### ISSUE THREAD

**Issue by alcibiadeseth** [gakonst/foundry#169]
Right now, if multiple test files exist in test/, forge only executes the first one.

**Comment by mattsse** [comment]
this is with `forge test`, right?
expected behavior would be to execute everything under test

**Comment by alcibiadeseth** [comment]
Correct, it is only executing one file, and interestingly, that file changes on repeated invocations.

**Comment by alcibiadeseth** [comment]
```Athens % forge test
success.
Running 1 test for ContractTooTest
[PASS] testExample (gas: 254)
Athens % forge test
success.
Running 1 test for ContractTest
[PASS] testExample (gas: 254)

**Comment by alcibiadeseth** [comment]
I should add, this is on an M1 processor, if that would matter.

**Comment by alcibiadeseth** [comment]
I see this with only one file:

```Athens % forge test
no files changed, compilation skippped.
Running 1 test for "YapeswapTest.json":YapeswapTest
[PASS] testExample (gas: 254)

**Comment by gakonst** [comment]
Can you share the repo which causes this? I'll try to repro locally

**Comment by gakonst** [comment]
I think same fix from #170 applies to here, closing but please re-open if still an issue on latest master. Please also try sharing a repo which reproduces the issue, so we can debug it!
